### ISSUE TITLE
rviz2 plugin

### ISSUE THREAD

**Issue by fred-apex-ai** [ros-perception/radar_msgs#13]
Hey @SteveMacenski , do you know of any rviz2 plugin to visualize `RadarTracks` or `RadarScans`? Or how do you visualize these data? If there is none and I write one myself, what would be the appropriate place to share this code?

**Comment by SteveMacenski** [comment]
I don’t believe there are any rviz tools for these messages yet. In terms of where it should live, potentially it could live here or in another ros-perception repository (or your own github & I can help in the release process)

**Comment by fred-apex-ai** [comment]
@SteveMacenski Thanks. I will then work on an implementation myself and get back to you when it's ready for public use

**Comment by fred-apex-ai** [comment]
PUtting the rviz plugin here would make sense, just need to move the msg package into a separate subdirectory, nav2 does that, too

**Comment by stan-guer** [comment]
What's the status on this? Can someone else try to do this or will this be ready some time soon?
